parser.yy

```diff
@@ -270,7 +270,7 @@ Expression:
 |	Expression '-' Expression
 |	Expression '.' Expression
 |	Expression '/' Expression
-|	Expression ':' Expression
+|	PATTERN ':' Term
```

Comment:
As for the failing test, I still need to add the test corpus, I think. Perhaps the issue you just described and the failing test are related.